GitUI/Theming/Renderers/HeaderRenderer.cs

```diff
@@ -11,17 +11,16 @@ internal class HeaderRenderer : ThemeRenderer
         public override int RenderBackground(IntPtr hdc, int partId, int stateId, Rectangle prect,
             NativeMethods.RECTCLS pcliprect)
         {
-            using (var ctx = CreateRenderContext(hdc, pcliprect))
+            using var ctx = CreateRenderContext(hdc, pcliprect);
+            switch ((Parts)partId)
             {
-                switch ((Parts)partId)
-                {
-                    case Parts.None:
+                case Parts.None:
                     {
```

Comment:
Why not unindent this code block, too - and a few more below?

Reply:
I only left this to make sure someone reviewed - or it was something where autoindent is not working properly and I missed to to fix this manually after reverting.
Good catch, 
Pushed fixup.